BetaCameras/Xtion2/OpenNICAPI.cs

```diff
@@ -0,0 +1,478 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using Metrilus.Util;
+using System.Drawing;
+using System.Drawing.Imaging;
+
+namespace MetriCam2.Cameras
+{
+    public class OpenNICAPI
```

Comment:
Would be nice if a public class would have at least a comment about its purpose.

Reply:
Class name should be `OpenNI2CApi` (include version and follow C# naming conventions)

Reply:
done